### ISSUE TITLE
How to define publicPath?

### ISSUE THREAD

**Issue by wilsonpage** [arthurbergmz/webpack-pwa-manifest#10]
The injected HTML file path path doesn't seem to respect the [`publicPath`](https://webpack.js.org/configuration/output/#output-publicpath) option defined in my webpack config.

My Webpack config:

```js
output: {
  ...
  publicPath: '/static/',
  ...
},
...
```

Expected:

```html
<link rel="manifest" href="/static/manifest.593b44ef40ae0879f60c80e4156d2958.json">
```

Actual:

```html
<link rel="manifest" href="manifest.593b44ef40ae0879f60c80e4156d2958.json">
```

**Comment by arthurbergmz** [comment]
Added by #11

**Comment by arthurbergmz** [comment]
Available in **v3.1.0**.
Please, update and set `useWebpackPublicPath` as `true` in your _WebpackPwaManifest's options_.
